pkgs/applications/misc/prusa-slicer/default.nix

```diff
@@ -26,28 +29,50 @@
 , openvdb
 , pcre
 , qhull
-, tbb
-, wxGTK31
+, tbb_2021_8
+, wxGTK32
 , xorg
 , fetchpatch
 , withSystemd ? lib.meta.availableOn stdenv.hostPlatform systemd, systemd
 }:
 let
-  wxGTK-prusa = wxGTK31.overrideAttrs (old: rec {
+  wxGTK-prusa = wxGTK32.overrideAttrs (old: rec {
     pname = "wxwidgets-prusa3d-patched";
-    version = "3.1.4";
+    version = "3.2.0";
+    configureFlags = old.configureFlags ++ [ "--disable-glcanvasegl" ];
+    preConfigure = old.preConfigure + ''
+    sed -ie 's@^\t\(monodll_msw_secretstore\|monolib_msw_secretstore\|basedll_msw_secretstore\|coredll_dark_mode\|monodll_dark_mode\|monolib_dark_mode\|baselib_msw_secretstore\|corelib_dark_mode\)\.o$@\t\1.o \\@' Makefile.in
+    '';
     src = fetchFromGitHub {
       owner = "prusa3d";
       repo = "wxWidgets";
-      rev = "489f6118256853cf5b299d595868641938566cdb";
-      hash = "sha256-xGL5I2+bPjmZGSTYe1L7VAmvLHbwd934o/cxg9baEvQ=";
+      rev = "78aa2dc0ea7ce99dc19adc1140f74c3e2e3f3a26";
+      sha256 = "sha256-rYvmNmvv48JSKVT4ph9AS+JdstnLSRmcpWz1IdgBzQo=";
       fetchSubmodules = true;
     };
   });
+  nanosvg-fltk = stdenv.mkDerivation {
+    pname = "nanosvg-fltk";
+    version = "unstable-2022-12-22";
+
+    src = fetchFromGitHub {
+      owner = "fltk";
+      repo = "nanosvg";
+      rev = "abcd277ea45e9098bed752cf9c6875b533c0892f";
+      sha256 = "sha256-WNdAYu66ggpSYJ8Kt57yEA4mSTv+Rvzj9Rm1q765HpY=";
+    };
+
+    nativeBuildInputs = [
+      cmake
+    ];
+  };
+  openvdb_tbb_2021_8 = openvdb.overrideAttrs (old: rec {
+    buildInputs = [ openexr boost tbb_2021_8 jemalloc c-blosc ilmbase ];
+  });
```

Comment:
I get inlining the `wxGTK-prusa` package, but if you need to package a library, why not in `pkgs/development/libraries`? This also lacks `meta`.

Reply:
I've pulled it out into its own derivation. Is there a nicer way to specify vendored/patched packages?